.github/workflows/update_community_docs.yml

```diff
@@ -0,0 +1,40 @@
+name: Update community docs
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'Community/**.md'
```

Comment:
Let's add `Catalog/**.md` to paths. `library_criteria.md` to converted with this process as well